zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -88,7 +88,7 @@ public Factory(HttpClient httpClient) {
       } else {
         try {
           callback.onSuccess(parseResponse(response, bodyConverter));
-        } catch (IOException e) {
+        } catch (Throwable e) {
```

Comment:
usually I do either ` IOException|RuntimeException` or if doing Throwable, use propagateIfFatal

Reply:
I will scrub this in another pull request

Reply:
if you don't mind reverting this line, it won't drift :P

Reply:
Makes sense - going to stick with `Throwable` since I've seen a lot of our code, I think following okio's example, use `AssertionError` which would hang I think.

Reply:
Ok reverted